modules/common/services/audio.nix

```diff
@@ -77,6 +132,51 @@ in {
       script = ''${pkgs.pulseaudio}/bin/pa-info > /dev/null 2>&1'';
```

Comment:
Can we use lingering (see below) instead of the pulseaudio-starter service?

```
systemd.tmpfiles.rules = [
  "f /var/lib/systemd/linger/${config.ghaf.users.accounts.user}"
];
```